tests/e2e/utils/performance.js

```diff
@@ -0,0 +1,37 @@
+export async function getLoadingDurations() {
```

Comment:
Similar to the function from woocommerce-blocks https://github.com/woocommerce/woocommerce-blocks/blob/trunk/tests/utils/performance.ts. The differences are that I can't destruct the `performance.getEntriesByType(..)` assignment or use array prototype.find. I get the following errors:
`ReferenceError: _slicedToArray2` 
`ReferenceError: _find`

It could be a puppeteer problem since these are used within `page.evaulate()`. Our puppeteer version is 2 and it relies on [puppeteer-utils](https://github.com/Automattic/puppeteer-utils/blob/master/package.json#L26). Blocks uses [jest-environment-puppeteer](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/package.json#L178) where its puppeteer is at 14. It doesn't seem to have these issues. I think it would be good to upgrade it to the newer puppeteer version.

In any case, I worked around the `ReferenceError` and this function works in this branch.